[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:02,760 INFO [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:219] - [null, java.io.IOException: 远程主机强迫关闭了一个现有的连接。, java.io.IOException: 远程主机强迫关闭了一个现有的连接。] Unexpected exception during request: [null, java.io.IOException: 远程主机强迫关闭了一个现有的连接。, java.io.IOException: 远程主机强迫关闭了一个现有的连接。]
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:02,760 INFO [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:219] - [null, java.io.IOException: 远程主机强迫关闭了一个现有的连接。, java.io.IOException: 远程主机强迫关闭了一个现有的连接。] Unexpected exception during request: [null, java.io.IOException: 远程主机强迫关闭了一个现有的连接。, java.io.IOException: 远程主机强迫关闭了一个现有的连接。]
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:02,875 INFO [lettuce-eventExecutorLoop-1-10][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:02,875 INFO [lettuce-eventExecutorLoop-1-9][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:04,915 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:04,915 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:09,170 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:09,170 INFO [lettuce-eventExecutorLoop-1-15][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:11,196 WARN [lettuce-nioEventLoop-4-10][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:11,197 WARN [lettuce-nioEventLoop-4-9][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:15,563 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:15,563 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:17,596 WARN [lettuce-nioEventLoop-4-16][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:17,596 WARN [lettuce-nioEventLoop-4-15][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:22,764 INFO [lettuce-eventExecutorLoop-1-11][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:22,764 INFO [lettuce-eventExecutorLoop-1-12][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:24,790 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:24,790 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:29,967 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:29,967 INFO [lettuce-eventExecutorLoop-1-15][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:32,011 WARN [lettuce-nioEventLoop-4-10][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:32,011 WARN [lettuce-nioEventLoop-4-9][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:36,170 INFO [lettuce-eventExecutorLoop-1-16][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:36,170 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:38,199 WARN [lettuce-nioEventLoop-4-12][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:38,199 WARN [lettuce-nioEventLoop-4-11][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:46,471 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:46,471 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:48,513 WARN [lettuce-nioEventLoop-4-14][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:24:48,513 WARN [lettuce-nioEventLoop-4-13][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:25:04,964 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:25:04,964 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:25:07,005 WARN [lettuce-nioEventLoop-4-16][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:25:07,005 WARN [lettuce-nioEventLoop-4-15][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:25:37,066 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:25:37,066 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:25:37,068 INFO [lettuce-nioEventLoop-4-1][ReconnectionHandler.java:177] - Reconnected to localhost:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:25:37,068 INFO [lettuce-nioEventLoop-4-2][ReconnectionHandler.java:177] - Reconnected to localhost:6379
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:26:52,334 INFO [Thread-7][ExecutorConfigurationSupport.java:208] - Shutting down ExecutorService 'applicationTaskExecutor'
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:00,517 INFO [main][StartupInfoLogger.java:50] - Starting SailingBootstrap on LAPTOP-VBDG941V with PID 41968 (E:\MyCode\Java\sailing\target\classes started by liguohui in E:\MyCode\Java\sailing)
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:00,523 INFO [main][SpringApplication.java:675] - No active profile set, falling back to default profiles: default
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:01,093 INFO [main][RepositoryConfigurationDelegate.java:244] - Multiple Spring Data modules found, entering strict repository configuration mode!
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:01,095 INFO [main][RepositoryConfigurationDelegate.java:126] - Bootstrapping Spring Data repositories in DEFAULT mode.
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:01,116 INFO [main][RepositoryConfigurationDelegate.java:182] - Finished Spring Data repository scanning in 13ms. Found 0 repository interfaces.
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:01,458 INFO [main][TomcatWebServer.java:90] - Tomcat initialized with port(s): 56085 (http)
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:01,478 INFO [main][DirectJDKLog.java:173] - Initializing ProtocolHandler ["http-nio-56085"]
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:01,482 INFO [main][DirectJDKLog.java:173] - Starting service [Tomcat]
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:01,483 INFO [main][DirectJDKLog.java:173] - Starting Servlet engine: [Apache Tomcat/9.0.16]
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:01,487 INFO [main][DirectJDKLog.java:173] - The APR based Apache Tomcat Native library which allows optimal performance in production environments was not found on the java.library.path: [D:\JDK\jdk1.8.0_191\bin;C:\Windows\Sun\Java\bin;C:\Windows\system32;C:\Windows;D:\JDK\jdk1.8.0_191\bin;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\Program Files (x86)\VMware\VMware Workstation\bin\;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.0\bin;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.0\libnvvp;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.1\bin;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.1\libnvvp;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\Program Files\NVIDIA Corporation\Nsight Compute 2020.1.1\;D:\MinGW\bin;D:\ProgramData\Anaconda3;D:\ProgramData\Anaconda3\Scripts;D:\ProgramData\Anaconda3\Library\bin;D:\Program_My\mysql-8.0.27-winx64\bin;D:\JDK\jdk1.8.0_191\jre\bin;C:\Users\liguohui\AppData\Local\Microsoft\WindowsApps;;D:\Program Files\JetBrains\PyCharm 2021.2.3\bin;;D:\Program Files\JetBrains\IntelliJ IDEA 2021.2.3\bin;;D:\Program Files\JetBrains\CLion 2021.2.3\bin;;D:\Program Files\JetBrains\WebStorm 2021.2.3\bin;;.]
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:01,579 INFO [main][DirectJDKLog.java:173] - Initializing Spring embedded WebApplicationContext
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:01,579 INFO [main][ServletWebServerApplicationContext.java:296] - Root WebApplicationContext: initialization completed in 1006 ms
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:02,097 INFO [main][EndpointLinksResolver.java:59] - Exposing 2 endpoint(s) beneath base path '/actuator'
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:02,135 INFO [main][PropertySourcedRequestMappingHandlerMapping.java:69] - Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:02,202 INFO [main][ExecutorConfigurationSupport.java:171] - Initializing ExecutorService 'applicationTaskExecutor'
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:02,374 INFO [main][DocumentationPluginsBootstrapper.java:160] - Context refreshed
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:02,385 INFO [main][DocumentationPluginsBootstrapper.java:163] - Found 1 custom documentation plugin(s)
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:02,404 INFO [main][ApiListingReferenceScanner.java:41] - Scanning for api listing references
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:02,485 INFO [main][DirectJDKLog.java:173] - Starting ProtocolHandler ["http-nio-56085"]
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:02,498 INFO [main][TomcatWebServer.java:204] - Tomcat started on port(s): 56085 (http) with context path '/sailing'
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:02,500 INFO [main][StartupInfoLogger.java:59] - Started SailingBootstrap in 2.177 seconds (JVM running for 2.998)
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:03,093 INFO [RMI TCP Connection(2)-192.168.187.1][DirectJDKLog.java:173] - Initializing Spring DispatcherServlet 'dispatcherServlet'
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:03,093 INFO [RMI TCP Connection(2)-192.168.187.1][FrameworkServlet.java:524] - Initializing Servlet 'dispatcherServlet'
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:03,097 INFO [RMI TCP Connection(2)-192.168.187.1][FrameworkServlet.java:546] - Completed initialization in 4 ms
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:03,146 INFO [RMI TCP Connection(1)-192.168.187.1][EpollProvider.java:68] - Starting without optional epoll library
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:27:03,148 INFO [RMI TCP Connection(1)-192.168.187.1][KqueueProvider.java:70] - Starting without optional kqueue library
[${project.name}][${env:SERVER_PORT}] 2021-11-16 18:33:33,473 INFO [http-nio-56085-exec-1][QCloudSmsService.java:56] - 给手机号15279459350发送验证码：309239
